velox/functions/sparksql/Register.cpp

```diff
@@ -225,6 +225,11 @@ void registerFunctions(const std::string& prefix) {
   registerFunction<DayOfYearFunction, int64_t, Date>(
       {prefix + "doy", prefix + "dayofyear"});
 
+  registerFunction<DayOfWeekFunction, int64_t, Timestamp>(
```

Comment:
Return int32_t type.  The above function `dayofyear` return type is wrong. can you double check it?

Reply:
Fixed

Reply:
I used to want to be consistent with presto functions since int64_t can cover int32_t. It's also nice to keep consistent with the conventions in this module :)